R/summaries.R

```diff
@@ -321,23 +339,27 @@ setMethod("countDominantFeatures", signature = c(x = "SummarizedExperiment"),
     # Creates a tibble that contains number of times that a column of "name"
     # is present in samples and relative portion of samples where they
     # present.
+    
     if (is.null(group)) {
-        name <- sym(name)
+        colname <- sym(colname)
         data <- data %>%
-            group_by(!!name)
+            group_by(!!colname)
     } else {
         group <- sym(group)
-        name <- sym(name)
+        colname <- sym(colname)
         data <- data %>%
-            group_by(!!group, !!name)
+            group_by(!!group, !!colname)
     }
     tallied_data <- data %>%
         tally() %>%
         mutate(
-            rel.freq = round(100 * n / sum(n), 1),
-            rel.freq.pct = paste0(round(100 * n / sum(n), 0), "%")
+            rel.freq = n / sum(n)
         ) %>%
         arrange(desc(n))
+    if(!is.null(digits)) {
+      tallied_data["rel.freq"] <- round(tallied_data["rel.freq"], digits) 
```

Comment:
@antagomir SHould the `rel.freq` be `rel_freq`? By default, taxa column is named `dominant_taxa` (with underscore)

Or then we could change `dominant_taxa` to `dominant.taxa`. I think we do not have naming convention to tables

Reply:
Just letting you know that I've decided column name defaults to be with underscore and PR is ready for review if you agree

Reply:
nice!